Automattic/wp-calypso
# Stats: Update layout of commercial opt-out form

Related to #86834

## Proposed Changes

Moves the opt-out UI from the Panel interface into a new secondary Card interface. Always visible unless the site already has a paid commercial license.

### Before with opt-out inside a Panel control
<img width="585" alt="SCR-20240125-tgbr" src="https://github.com/Automattic/wp-calypso/assets/40267301/081fc115-f43a-4f65-800e-543a6efff6dc">

### After with opt-out inside a Card interface
<img width="582" alt="SCR-20240125-tgie" src="https://github.com/Automattic/wp-calypso/assets/40267301/3ab91eb5-2fe3-4113-bb64-647c11556132">

### Variant for sites already classified as "commercial"
<img width="471" alt="SCR-20240125-thle" src="https://github.com/Automattic/wp-calypso/assets/40267301/ca775d4e-e167-4b23-9a20-27d49b2c314e">

## Testing Instructions



* Launch the Calypso Live link.
* Navigate to the Stats page and click the upgrade banner.
* If you are shown the PWYW UI, click the link to switch to a commercial license.
* Enable the flag as needed: `stats/checkout-flows-v2`
* Confirm the UI matches the above screenshot. No Panel dropdown control, new card UI, button is visible even if checkboxes are unchecked.
* Check all checkboxes and confirm button is enabled.
* Click the button to jump to the PWYW interface.

**Note:** If your site has already been classified as commercial, you'll be prompted to open your email client with a prefilled email title/body provided. In this scenario, the info text and button text will be different (see variant screenshot).

## Pre-merge Checklist
